grr/server/grr_response_server/gui/ui/components/client/module.ts

```diff
@@ -27,6 +28,7 @@ import {TimestampModule} from '../timestamp/module';
   ],
   declarations: [
     Client,
+    StatusChip,
```

Comment:
Please move StatusChip with all its files in a dedicated directory, e.g. `[...]/ui/components/<component-name>/` and create a dedicated Module only for StatusChip.

Reply:
Alright!